### ISSUE TITLE
Add serialized name of a property/class to documentation

### ISSUE THREAD

**Issue by bact** [spdx/spdx-spec#975]
Currently a TODO in Getting started

https://github.com/spdx/spdx-spec/blob/aac3e38b0b01b2ccb129367617a07598cf0ccdd8/docs/annexes/getting-started.md?plain=1#L391-L392

**Comment by bact** [comment]
Proposed code are now available at repos as shown in https://github.com/spdx/spec-parser/issues/132 but still need to decide about where should this serialized name being displayed. 

Two options are shown here for discussion, more can be added:

1. Display the serialized name just immediately after the name<br />
  <img width="400" src="https://github.com/spdx/spdx-spec/assets/128572/33462715-b83e-4351-8ac6-acdbf3cfc19a"><br /><img width="500" src="https://github.com/spdx/spdx-spec/assets/128572/ee249761-34cb-4bee-a89f-901b2075856e">
2. Display the serialized name inside the Metadata box<br />
  <img width="250" src="https://github.com/spdx/spdx-spec/assets/128572/0cac28a5-9710-407f-8f6b-6802fa5fd93a"><br /><img width="350" src="https://github.com/spdx/spdx-spec/assets/128572/b060eaed-2e67-48d4-b8e4-aa6b986f58e0">

**Comment by zvr** [comment]
I don't believe this should be visible there.

It's part of the JSON-LD context, and it is relevant only for the JSON-LD serialization.

We could have a short sentence explaining its format in the [serializations](https://github.com/spdx/spdx-spec/blob/development/v3.0.1/docs/serializations.md) chapter.

**Comment by bact** [comment]
I agree that it's only relevant in the JSON-LD context and not for other serialization options, so it can be a bit "intrusive" to include this information in the main doc (which meant to be generic/neutral).

But still believe that it should be generated and visible somewhere for easy reference, providing more confidence for the SBOM author when authoring/checking SBOM.

**Comment by bact** [comment]
- Updated the issue title - to ask for comments
- Updated the issue description - add considerations to be made

**Comment by bact** [comment]
During 2024-07-09 Tech team meeting, towards the very end in the context of an updated spec parser, there is a discussion about the availability of this serialized name. There's no conclusion yet. From the minutes:

```text
- Is it possible to put the serialized JSON property in the table?
  - Joshua thinks that this will help with confusion.
    - Document of JSON-LD context needs to be somewhere.
    - Writing a JSON-LD document from the page won't work.
  - Alexios notes that the document describes RDF model.
```

**Comment by goneall** [comment]
Per comment in SPDX minutes - moving this to post 3.0.1
